fdbbackup/backup.actor.cpp

```diff
@@ -100,6 +101,12 @@ enum class DBType { UNDEFINED = 0, START, STATUS, SWITCH, ABORT, PAUSE, RESUME }
 // New fast restore reuses the type from legacy slow restore
 enum class RestoreType { UNKNOWN, START, STATUS, ABORT, WAIT };
 
+enum class DataMovementType{
+	UNDEFINED = 0,
+	START, STATUS, SWITCH, ABORT, PAUSE, RESUME
+	// todo: add more operation type
```

Comment:
Nit: usually when we use todos, we capitalize them as such:

```suggestion
	// TODO: add more operation type
```